src/Tribe/Timezones.php

```diff
@@ -0,0 +1,374 @@
+<?php
+
+/**
+ * Helpers for handling timezone based event datetimes.
+ *
+ * In our timezone logic, the term "local" refers to the locality of an event
+ * rather than the local WordPress timezone.
+ */
+class Tribe__Events__Timezones {
+	const SITE_TIMEZONE  = 'site';
+	const EVENT_TIMEZONE = 'event';
+
+
+	/**
+	 * Container for reusable DateTimeZone objects.
+	 *
+	 * @var array
+	 */
+	protected static $timezones = array();
+
+
+	public static function init() {
+		self::display_timezones();
+	}
+
+	/**
+	 * Takes care of appending timezone information to the display of
+	 * event date/times.
+	 */
+	protected static function display_timezones() {
+		if ( tribe_get_option( 'tribe_events_timezones_show_zone' ) ) {
+			add_filter( 'tribe_events_event_schedule_details', array( __CLASS__, 'append_timezone' ), 10, 2 );
```

Comment:
Appending the timezone using this filter causes the timezone to be inserted after the `$before`/`$after` values passed in to the `tribe_events_event_schedule_details()` function:

This (using one of my events on my local instance):

```
tribe_events_event_schedule_details( $event, '<h3>', '</h3>' );
```

Would result in:

```
<h3>
  <span class="date-start dtstart">
    August 6@3:00 pm
    <span class="value-title" title="2015-08-06PDT03:00"></span>
  </span>
  - 
  <span class="end-time dtend">
    8:00 pm
    <span class="value-title" title="2015-08-06PDT08:00"></span>
  </span>
</h3>
<span class="timezone"> PDT </span>
```

Which results in some funkiness with the appearance:

![screen shot 2015-08-05 at 4 41 51 pm](https://cloud.githubusercontent.com/assets/430385/9097322/ecb43080-3b90-11e5-82d3-657ac74cbadc.png)

Perhaps we need to add a new filter [around `$schedule`](https://github.com/moderntribe/the-events-calendar/blob/release/120/src/functions/template-tags/general.php#L1116) and use that filter instead?